proto/netbookai/spawner/spawner.proto

```diff
@@ -569,3 +571,15 @@ message CopySnapshotResponse {
   string newSnapshotId = 1;
   string newSnapshotUri = 2;
 }
+
+message PresignS3UrlRequest {
+  string region = 1;
+  string accountName = 2;
+  string bucket = 3;
+  string file = 4;
```

Comment:
Can we call this filePath? As we could be using a wildcard path to get everything in a folder.

Reply:
no, it has to the object key. The object being here the file in the bucket. Let me see if we can specify the folder name